packages/modules/src/modules/banners/usEoyMoment/UsEoyMomentGivingTuesdayBanner.tsx

```diff
@@ -0,0 +1,12 @@
+import { bannerWrapper, validatedBannerWrapper } from '../common/BannerWrapper';
+import getUsEoyMomentBanner from './components/getUsEoyMomentBanner';
+
+const UsEoyMomentGivingTuesdayBanner = getUsEoyMomentBanner(true);
+
+const unvalidated = bannerWrapper(UsEoyMomentGivingTuesdayBanner, 'us-eoy-moment-banner');
+const validated = validatedBannerWrapper(UsEoyMomentGivingTuesdayBanner, 'us-eoy-moment-banner');
```

Comment:
might be worth updating this to `us-eoy-moment-giving-tuesday-banner` - it just affects the `componentId`s for ophan but might be nice to be able to distinguish GT banners from the regular ones